src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,63 @@
+---
+authors:
+  - "alearori#8661"
+created_at: 2019/11/15
+title: "Multidimensional Arrays"
+---
+
+## What is a multidimensional array?
+
+To put simply, a multidimensional array is an array that contains more than one row and column. It is commonly used for making matricies, but it can also be used for storing information such as if you use `string`.
+
+### It is NOT an array inside of an array
+
+It just looks like one. Unlike with other container types, an array cannot hold anything other than the original data type, so if you declare it to be of data type `int` at the start, it cannot hold anything else inside. If you're looking to use multiple data types, you should make a `class`, `struct` or use other container types like `map` that fulfill your requirement.
+
+### How do we make one?
+
+To create multidimensional array, you do it like a regular array but with a extra `[]` at the end. 
+
+#### Method 1: 
+Use `data_type array_name[A][B]` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `A` and `B` are the integers.
+
+Example: `int arr[2][3]` will make a empty 2x3 matrix that looks like: `{{0,0,0},{0,0,0}}`
+
+#### Method 2:
+Use `data_type array_name[][] = {{a1,a2,a3,...},{b1,b2,b3,...}}` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `a1, a2, a3, b1, b2, b3,...` are objects of the data type that was chosen at the start in `data_type`, but the number of `a`'s and `b`'s must be equal.
+
+Example: `int arr[][] = {{1,2,3}, {4,5,6}}` will make a 2x3 matrix that looks like: `{{1,2,3},{4,5,6}}`
+
+#### Method 3:
+
+Combine both `data_type array_name[A][B] = {{a1,a2,a3,...},{b1,b2,b3,...}}` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `a1, a2, a3, b1, b2, b3,...` are objects of the data type that was chosen at the start in `data_type` and `A` and `B` is the number of `{}`'s and `a`'s and `b`'s respectively.
+
+Example: `int arr[2][3] = {{1,2,3}, {4,5,6}}` will make a 2x3 matrix that looks like: `{{1,2,3},{4,5,6}}`.
+
+#### Note:
+It is possible to go ever further, and so something like `data_type array_name[A][B][C]` is possible and would add "height" to your array. 
+
+For example, if you do: `int arr[2][3][4]`, you'll make an array that looks like: `{{{0,0,0,0},{0,0,0,0},{0,0,0,0}},{{0,0,0,0},{0,0,0,0},{0,0,0,0}}}`
+
+### What can you do with it?
+
+Everything you can do with a one dimensional array.
+
+#### Calling objects
+To get the thing inside, you can call for the thing inside a multidimensional array by putting a integer inside the `[]`. 
```

Comment:
Would be better to use a more descriptive word than "thing". Other options could be "element", "data", "value"

Reply:
Okay, I expanded on what "things" could be, and updated the pull request.